rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -0,0 +1,632 @@
+= Design Proposal: Pablo Fees & Staking Rewards Distribution
+:math:
+:stem:
+:imagesoutdir: images
+:imagesdir: images
+:toc:
+:toclevels: 4
+:sectnums:
+:sectnumlevels: 4
+
+== Abstract
+
+This document proposes the Pablo distribution(token and pool trading fees) mechanism while considering various options and capturing the discussions about the subject.
+
+`TODO summarise the mechanism`
+
+== Background
+
+=== PBLO Token Initial Distribution
+
+According to the http://link[tokenomics design] the Pablo is supposed to be distributed as follows,
+
+* Ecosystem fund - x
+* Farming rewards - y
+* LBP Auction - z
+* ...
+
+TODO fill above with details from the original sheet.
+
+The farming rewards are incentives for liquidity provider(LP)s who stake their LP tokens for Pablo pools. How much reward is allocated as incentive for each pool is to be decided by governance.
+
+=== Pool Fees
+
+Composable intends to distribute some percentage of the swap(transaction) fees captured by the pools in Pablo dex pallet as rewards to users who stake their `PBLO` tokens using the staking-rewards pallet interface. The idea is to incentivize the continuous owning of the staked `PICA` and `PBLO` to earn these yields which increases the value of the ecosystem overall by increasing the desirability of the staked assets.
+
+At the time of writing Pablo has the following fee parameters other than for liquidity bootstrapping pools(LBP) which do not charge fees,
+
+. LP Fee - A percentage of the trading fee that is distributed to liquidity providers based on the number of liquidity provider(LP) tokens they minted at the time of providing the liquidity.
+. Pool Owner Fee - A percentage of the trading fee that is distributed to the pool owner.
+
+==== LP Fee Distribution
+
+This is yet to be implemented in Pablo, hence the idea is that it can be addressed in the context of this proposal.
+
+== Use Cases
+
+Following is a summary of use cases omitting the UI specific use cases for brevity.
```

Comment:
you may omit ommitting notes for bravity too

Reply:
I'm not sure I follow what you mean? Could you clarify or give an example?

Reply:
remove this text

Reply:
Why should it be removed? It's providing context that UI specific use cases are not covered by this doc.